File: 2.0/java/media/getMedia.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.callstexts.model.Media;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        // you can limit what fields will be returned in second argument
        Media media = client.mediaApi().get(379506003L, "name,publicUrl");
    }
}

File: 2.0/csharp/media/getMedia.cs
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        Media media = client.MediaApi.Get(379506003, "name,publicUrl");
    }
}
